apps/frontend/src/pages/auth/components/sign-up-form/sign-up-form.tsx

```diff
@@ -18,36 +21,76 @@ const SignUpForm = ({ onSubmit }: Properties): JSX.Element => {
 		validationSchema: userSignUpValidationSchema,
 	});
 
+	const [isPasswordVisible, setIsPasswordVisible] = useState<boolean>(false);
+	const navigate = useNavigate();
+
 	const handleFormSubmit = useCallback(
 		(event_: React.BaseSyntheticEvent): void => {
-			void handleSubmit(onSubmit)(event_);
+			void handleSubmit(async (formData: UserSignUpRequestDto) => {
+				await onSubmit(formData);
+				navigate(AppRoute.ROOT);
+			})(event_);
 		},
-		[handleSubmit, onSubmit],
+		[handleSubmit, onSubmit, navigate],
 	);
 
+	const handleTogglePasswordVisibility = useCallback(() => {
+		setIsPasswordVisible((previousState) => !previousState);
+	}, []);
```

Comment:
Couldn't we use here `setIsPasswordVisible(!isPasswordVisible);`

Reply:
Some problems with ESLint. https://stackoverflow.com/questions/36677733/why-shouldnt-jsx-props-use-arrow-functions-or-bind

Reply:
Then let's leave it as it is